crates/pallet-feeds/src/feed_processor.rs

```diff
@@ -35,15 +35,28 @@ pub type FeedMetadata = Vec<u8>;
 pub trait FeedProcessor<FeedId> {
     /// initiates a specific Feed with data transparent to FeedProcessor
     /// can be called when re-initializing the feed.
-    fn init(&self, feed_id: FeedId, data: &[u8]) -> DispatchResult;
+    fn init(&self, _feed_id: FeedId, _data: &[u8]) -> DispatchResult {
```

Comment:
Should user always specify `init`, `put`, `delete`, and `object_mappings`?

Reply:
no required. Take a look at ParachainImported. It only specifies a few of these